numhask-range-0.2.2.1

As seen on the build server:

```
Building library for numhask-range-0.2.2.1..
[1 of 6] Compiling NumHask.Pair     ( src/NumHask/Pair.hs, dist/build/NumHask/Pair.o )

src/NumHask/Pair.hs:213:31: error:
    • Could not deduce (AdditiveInvertible a) arising from a use of ‘-’
      from the context: Epsilon a
        bound by the instance declaration at src/NumHask/Pair.hs:211:10-40
      Possible fix:
        add (AdditiveInvertible a) to the context of
          the instance declaration
    • In the second argument of ‘($)’, namely ‘a - b’
      In the expression: nearZero $ a - b
      In an equation for ‘aboutEqual’: aboutEqual a b = nearZero $ a - b
    |
213 |   aboutEqual a b = nearZero $ a - b
    |                               ^^^^^
```

CC @tonyday567

--- a/build-constraints.yaml
+++ b/build-constraints.yaml
@@ -3551,12 +3551,6 @@ packages:
         # https://github.com/commercialhaskell/stackage/issues/3728
         - aeson < 1.4
 
-        # https://github.com/commercialhaskell/stackage/issues/3742
-        - numhask < 0.2.3
-        - numhask-prelude < 0.1.0.0
-        - numhask-range < 0.2.3.1
-        - numhask-test < 0.1.0.0
-
         # https://github.com/commercialhaskell/stackage/issues/3750
         - servant < 0.14
         - servant-client < 0.14

